tests/context.py

```
@@ -83,44 +83,16 @@
-		unittest = sys.modules["""unittest"""]
-except ImportError as err:  # pragma: no branch
-	raise ModuleNotFoundError("[CWE-440] unittest Failed to import.") from err
-
-try:
-	if 'contextlib' not in sys.modules:
-		import contextlib
-	else:  # pragma: no branch
-		contextlib = sys.modules["""contextlib"""]
-except ImportError as err:  # pragma: no branch
-	raise ModuleNotFoundError("[CWE-440] contextlib Failed to import.") from err
+	raise ModuleNotFoundError("[CWE-440] Module Failed to import.") from err
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)